src/modules/controllers/task.controllers.js

```diff
@@ -0,0 +1,65 @@
+const tasksSchema = require('../../schema/taskSchema')
+
+module.exports.getAllTasks = async (req, res) => {
+  try {
+    const allTasks = await tasksSchema.find()
+    res.send(allTasks)
+  } catch (e) {
+    res.status(500).send({ message: 'Internal server error' });
+  }
+};
+
+module.exports.createNewTask = async (req, res) => {
+  try {
+    const { isCheck, text } = req.body
+    console.log(req.body)
+    if(isCheck !== undefined && text){
+      const newTask = new tasksSchema({
+        text: text,
+        isCheck: isCheck
```

Comment:
it is  object. if key name and key value is the same, u must wrote only key name